sdk/core/Azure.Core/src/Shared/PemReader.cs

```diff
@@ -12,6 +12,12 @@ namespace Azure.Core
     /// <summary>
     /// Reads PEM streams to parse PEM fields or load certificates.
     /// </summary>
+    /// <remarks>
+    /// This class provides a downlevel PEM decoder since <c>PemEncoding</c> wasn't added until net5.0.
+    /// The <c>PemEncoding</c> class takes advantage of other implementation changes in net5.0 and,
+    /// based on conversations with the .NET team, runtime changes. This was based on <c>PemEncodiner</c>
```

Comment:
```suggestion
    /// based on conversations with the .NET team, runtime changes. This was based on <c>PemEncoding</c>
```

Reply:
Ugh. I kept wanting to type PemEncoder even though I knew it was PemEncoding. Guess I got my wires crossed here.